### ISSUE TITLE
Fix ABC local playback

### ISSUE THREAD

**Issue by eracknaphobia** [eracknaphobia/plugin.video.fubotv#2]

**Comment by eracknaphobia** [comment]
Hmm, not much sticking out to me there. I'll send you a DM through the forums to see if you can get me the request from your browser in dev mode.

**Comment by eracknaphobia** [comment]
@usphil Please try version 2024.4.12 and see if it works for you.

**Comment by eracknaphobia** [comment]
Both abc and disney play for me....

Looks like you are running on 32bit arm device. I've mainly be working with 64bit windows and macos versions of kodi. Can you test on another device?

![image](https://github.com/eracknaphobia/plugin.video.fubotv/assets/8268725/b915ee49-1673-45fb-a642-b3fab3752de7)

**Comment by usphil** [comment]
My device is 64bit Amlogic. I will try it on Windows Kodi, then let you know.

**Comment by eracknaphobia** [comment]
Oh gotcha, sorry i was looking at the kodi version.

```vb
Starting Kodi (20.1 (20.1.0) Git:fccf3356b956bf7d3f12cab98626d5d0d4411f7f). Platform: Linux ARM 32-bit
Using Release Kodi x32
Kodi compiled 2023-03-16 by GCC 12.2.0 for Linux ARM 32-bit version 4.9.255 (264703)
Running on Amlogic with CoreElec X96X9: 20.3, kernel: Linux ARM 64-bit version 4.9.269
```

Also, your on inputstream.adaptive 20.3.7.1 the lastest for Nexus is 20.3.18
I'd try updating at least inputstream.adaptive and see if that fixes your issue.

**Comment by usphil** [comment]
That's right, I also think it's related to inputstream.adaptive. Because on Windows Kodi Nexus they can be played
